Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -371,8 +375,8 @@ private void InitializeInstance()
                 {
                     if (instance != null)
                     {
-                        Debug.Assert(instance.transform.parent == null, "The MixedRealityToolkit should not be parented under any other GameObject!");
-                        Debug.Assert(instance.transform.childCount == 0, "The MixedRealityToolkit should not have GameObject children!");
+                        DebugUtilities.Assert(instance.transform.parent == null, "The MixedRealityToolkit should not be parented under any other GameObject!");
```

Comment:
I hope we don't have to go through each feature area and replace all our debug statements.

I'd almost want to say each feature area should have it's own logging level, so some can be more verbose than others.

Reply:
> I hope we don't have to go through each feature area and replace all our debug statements.

We do because static classes (ex: UnityEngine.Debug) cannot have extension methods. I am signing myself up to update the calls.

> I'd almost want to say each feature area should have it's own logging level, so some can be more verbose than others.

The implementation makes that very doable. Each system would use the DebugUtilities overload that takes in a LoggingLevels parameter.